terraform/variables.tf

```diff
@@ -75,6 +75,10 @@ variable "forms_api_url" {
   type    = string
   default = "https://api.resourcewatch.org/v1"
 }
+variable "alerts_api_url" {
+  type    = string
+  default = https://data-api.globalforestwatch.org/
```

Comment:
This needs to be in quotes "..."

Plus, need to remove the ending `/` to avoid `//` issues with API calls